CRUDRecipeEF.BL/Services/IRecipeService.cs

```diff
@@ -18,5 +19,6 @@ public interface IRecipeService
         Task RemoveIngredientFromRecipe(string ingredientName, string recipeName);
 
         Task DeleteRecipe(string name);
+        Task UpdateRecipe(RecipeDTO recipe, string newName);
```

Comment:
take in a RecipeDTO, string recipeName since we use the recipe name to find the recipe in the db and they maybe changed the name of the one on the DTO